src/interfaces/mining.h

```diff
@@ -64,6 +64,9 @@ class BlockTemplate
      *                      for the next block should rise (default infinite).
```

Comment:
In "On testnet create min diff template after 20 mins" 6bf77e5a376804c0681897fa032537ba16e9a99a

nit: 
@Sjors should this commit message and comments be updated to reflect the actual change something like

"miner: update `waitNext` to return a new block if 20 minutes have elapsed since the tip"

I think the above discription is exactly what this commit does and does not enable the creation of the min diff template, it's done internally by the block assembler?

Reply:
Changed the commit message.